Source/Cake.Coveralls.Tests/Cake.Coveralls.Tests.csproj

```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;net6.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
```

Comment:
Was there a specific reason for not including the `net8.0` framework in the release?

Reply:
The understanding is that `net8.0` will be included in the [v4.0.0 release](https://www.nuget.org/packages/Cake.Core/4.0.0#supportedframeworks-body-tab) of the Core, right?

Reply:
> Was there a specific reason for not including the `net8.0` framework in the release?

Yes-and-no. I always try to keep the TMFs for the tests close to the TFMs for the "real" package. `Cake.Coveralls.csproj` also lists  `net6.0;net7.0` as TFMs.

> The understanding is that `net8.0` will be included in the [v4.0.0 release](https://www.nuget.org/packages/Cake.Core/4.0.0#supportedframeworks-body-tab) of the Core, right?

Correct. According to [CakeContrib.Guidelines#TargetFramework](https://cake-contrib.github.io/CakeContrib.Guidelines/guidelines/TargetFramework) addins targeting Cake v4.0.0 should use `net6.0;net7.0;net8.0` as TFMs.

Reply:
Thank you for the explanations! The [net8.0](https://github.com/cake-contrib/Cake.Coveralls/pull/124/files#diff-ca729cd1a5d9ff1740b8b58738ee274c6ccd09a2e73d0ccba1f99f05c5a4c7c2R4) TFM has been added in PR #124.